src/lsg/patterns/add-button/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { colors } from '../colors';
+import { Icon, IconName, Size as IconSize } from '../icons';
+import * as React from 'react';
+import { getSpace, Size as SpaceSize } from '../space';
+import styled from 'styled-components';
+
+export interface AddButtonProps {
+	onClick?: React.MouseEventHandler<HTMLElement>;
+	label?: string;
+	active?: boolean;
+}
+
+interface StyledAddButtonProps {
+	active?: boolean;
+}
+
+interface StyledIconProps {
+	active?: boolean;
+}
+
+const StyledAddButton = styled.div`
+	position: absolute;
```

Comment:
Should be `position: fixed`, currently this happens: 
<img width="704" alt="screen shot 2018-04-25 at 18 42 08" src="https://user-images.githubusercontent.com/4248851/39260844-f0a81ba4-48ba-11e8-9528-ed8d3082b98e.png">

Reply:
Good catch, thanks!